code/game/objects/effects/anomalies.dm

```diff
@@ -221,12 +223,13 @@
 
 /obj/effect/anomaly/flux/detonate()
 	if(explosive)
-		explosion(src, 1, 4, 16, 18) //Low devastation, but hits a lot of stuff.
+		explosion(src, 1, 4, 16, 18, FALSE) // Set adminlog to FALSE for custom logging
+		if(drops_core)
+			message_admins("A [name] has detonated at [impact_area][ADMIN_COORDJMP(impact_area)]")
+			log_admin("A [name] has detonated at [impact_area]")
```

Comment:
```suggestion
		message_admins("A [name] has detonated at [impact_area][ADMIN_COORDJMP(impact_area)]")
		log_admin("A [name] has detonated at [impact_area]")
```
Explosive flux anomalies should always be logged
If they don't drop a core they aren't explosive (I hope)